ground/openpilotgcs/src/plugins/config/cfg_vehicletypes/configmultirotorwidget.cpp

```diff
@@ -787,7 +787,7 @@ QString ConfigMultiRotorWidget::updateConfigObjectsFromWidgets()
 
             // Tricopter : Yaw mix slider value applies to servo (was fixed)
             // Get absolute MixerValueYaw, no servo reverse when Reverse All Motors is checked
-            setMixerVectorValue(mixer, channel, VehicleConfig::MIXERVECTOR_YAW, abs(getMixerValue(mixer, "MixerValueYaw")) * 1.27);
+            setMixerVectorValue(mixer, channel, VehicleConfig::MIXERVECTOR_YAW, abs(int(getMixerValue(mixer, "MixerValueYaw"))) * 1.27);
```

Comment:
I prefer keep abs/fabs here, i spent some time to avoid strange mixer results when you look at full mixer table and reliable values when board is disconnected / connected. ;)